inc/taskjob.class.php

```diff
@@ -530,15 +447,9 @@ function dropdownAction($myname, $actiontype, $method, $actiontypeid, $value=0,
     * @return array [id integed agent id] => $name value agent name
     */
    function getAgents($module) {
-
-      $array = [];
-      $array[".1"] = " ".__('Auto managenement dynamic of agents', 'fusioninventory');
-
-      $array[".2"] = " ".__('Auto managenement dynamic of agents (same subnet)', 'fusioninventory');
-
+      $array         = [];
       $pfAgentmodule = new PluginFusioninventoryAgentmodule();
-      $array1 = $pfAgentmodule->getAgentsCanDo(strtoupper($module));
```

Comment:
Hum... Maybe the original way was better for perfs

Reply:
you're right I've reverted this modification